pxr/base/tf/testenv/weakPtr.cpp

```diff
@@ -124,6 +124,24 @@ static void _TestComparisons()
     TF_AXIOM( !(NULL == x) );
 }
 
+static void _TestHash(){
+    TF_AXIOM(TfHash()(MonkeyInterfaceWeakPtr()) == TfHash()(MonkeyInterfaceWeakPtr(nullptr)));
+
+    {
+        Human h;
+        MonkeyInterfaceWeakPtr p(&h);
+        TF_AXIOM(TfHash()(p) == TfHash()(p));
+        TF_AXIOM(TfHash()(p) == TfHash()(MonkeyInterfaceWeakPtr(p)));
+    }
+    {
+        // Verify collisions are unlikely
+        Human h1, h2;
+        MonkeyInterfaceWeakPtr p1(&h1), p2(&h2);
+        TF_AXIOM(TfHash()(p1) != TfHash()(p2));
```

Comment:
It doesn't seem robust/possible to verify that something is "unlikely".  Unless we are 100% sure (or are making a defining statement) that equal hashes are **impossible** here, then the test could fail randomly, and we suggest removing it.

Reply:
This makes sense to remove.  Running that test I think was necessarily for validation, but I agree it's not obvious how to include it in the unit test.